phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -728,6 +729,13 @@ private static int getReservedQualifier(byte[] bytes, int offset, int length) {
      * (use @getPhysicalTableName for this case) 
      */
     PName getParentTableName();
+
+    /**
+     * @return the logical name of the parent. In case of the view index, it is the _IDX_+logical name of base table
```

Comment:
why _IDX_ + logical name of base table for view index? That's the physical table the view index is stored in, not the logical name.

Reply:
I am not sure I understand your comment. IDX prefix + logical name of the table is where the view index is stored in.